flow/core/kernel/vehicle/aimsun.py

```diff
@@ -119,15 +119,16 @@ def initialize(self, vehicles):
                 veh_id = '{}_{}'.format(typ['veh_id'], i)
                 self.__vehicles[veh_id] = dict()
                 self.__vehicles[veh_id]['type'] = typ['veh_id']
+                self.__vehicles[veh_id]['type_name'] = typ['veh_id']  # FIXME
                 self.__vehicles[veh_id]['initial_speed'] = typ['initial_speed']
                 self.num_vehicles += 1
                 if typ['acceleration_controller'][0] == RLController:
                     self.num_rl_vehicles += 1
 
-        for tracked_type in self.tracked_vehicle_types:
-            self.num_type[tracked_type] = 0
-            self.total_num_type[tracked_type] = 0
-            self.type_parameters[tracked_type] = {}
+        # for tracked_type in self.tracked_vehicle_types:
+        #     self.num_type[tracked_type] = 0
+        #     self.total_num_type[tracked_type] = 0
+        #     self.type_parameters[tracked_type] = {}
```

Comment:
why do we need to comment these lines? not disapproving, just curious

Reply:
The bug here is that it assigns an empty dict to type_parameters. Since I commented these lines, now the load.py doesn't work. Nathan is working on it.